Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptSearchController.java

```diff
@@ -87,6 +87,8 @@ public String search(HttpServletRequest req, ModelMap model, @RequestParam(defau
         if (results.hasErrors()) {
             return "conceptsearch";
         }
+        String str = conceptSearchBean.getWord().trim();
+        conceptSearchBean.setWord(str);
```

Comment:
merge into one line

Reply:
merged to single line and committed the change to the right branch named bug/CCP-243. Sorry about all the confusion.